tests/python/pants_test/backend/python/tasks/test_resolve_requirements.py

```diff
@@ -34,7 +35,12 @@ def test_resolve_simple_requirements(self):
     # the requirement isn't sneaking in some other way, which would render the remainder
     # of this test moot.)
     _, stderr_data = self._exercise_module(self._resolve_requirements([noreqs_tgt]), 'colors')
-    self.assertIn('ImportError: No module named colors', stderr_data)
+
+    try:
+      self.assertIn("ModuleNotFoundError: No module named 'colors'", stderr_data)
+    except AssertionError:
```

Comment:
I went with `try` construct here because this isn't Py2 vs Py3, it's Py3.6+ vs everything else, and `future.utils` doesn't have a `PY36` value and using `sys.info` doesn't read very well.